examples/3d_rendering_processing/transform_feedback.lpr

```diff
@@ -0,0 +1,135 @@
+{
+  Copyright 2021 Trung Le (Kagamma).
+
+  This file is part of "Castle Game Engine".
+
+  "Castle Game Engine" is free software; see the file COPYING.txt,
+  included in this distribution, for details about the copyright.
+
+  "Castle Game Engine" is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+
+  ----------------------------------------------------------------------------
+}
+
+{ Example how to use TGLSLProgram.SetTransformFeedbackVaryings }
+
+program transform_feedback;
+
+{$macro on}
+{$define nl:=+ LineEnding +}
```

Comment:
Please use `NL` constant from `CastleUtils` instead, without a macro, like 

```pascal
`xxx` + NL +
...
```

It looks only a little longer than your

```pascal
`xxx`nl
...
```

but it more obvious and consistent with other CGE code.